sdk/core/azure-core-opentelemetry/src/private/package_version.hpp

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+/**
+ * @file
+ * @brief Provides version information.
+ */
+
+#pragma once
+
+#include <cstdint>
+
+#define AZURE_CORE_OPENTELEMETRY_VERSION_MAJOR 1
+#define AZURE_CORE_OPENTELEMETRY_VERSION_MINOR 0
+#define AZURE_CORE_OPENTELEMETRY_VERSION_PATCH 0
+#define AZURE_CORE_OPENTELEMETRY_VERSION_PRERELEASE "beta.1"
+
+#define AZURE_CORE_OPENTELEMETRY_VERSION_ITOA_HELPER(i) #i
+#define AZURE_CORE_OPENTELEMETRY_VERSION_ITOA(i) AZURE_CORE_OPENTELEMETRY_VERSION_ITOA_HELPER(i)
+
+namespace Azure { namespace Core { namespace OpenTelemetry { namespace _detail {
+  /**
+   * @brief Provides version information.
+   *
+   */
+  class PackageVersion final {
+  public:
+    /// Major numeric identifier.
```

Comment:
not the right comment type 
use 
/**
* @brief
*/

Reply:
I'm following the example used in template. If this changes, we should change all the packages that follow the template's example.

Reply:
yet somehow it is incosistent with itself the class comment is one way, the public members is another. also if i remeber corectly there is a guidance for cpp regarding comments on public members https://azure.github.io/azure-sdk/cpp_introduction.html#documentation-style

Reply:
I agree that this should be fixed. But fixing it involves changing:
* Azure::Attestation
* Azure::Core
* Azure::Identity
* Azure::Keyvault::Secrets
* Azure::Keyvault::Cryptography
* Azure::Keyvault::Certificates
* Azure::Storage::Blobs
* Azure::Storage::Common
* Azure::Storage::Files::Datalake
* Azure::Storage::Files::Shares
* Azure::Storage::Queues
* Azure::Template

Making that change is beyond the scope of this PR. However it makes a lot of sense long-term. I'll raise an issue.